reference/customers.v3.yml

```diff
@@ -242,6 +242,9 @@ paths:
         * Attributes must be [created](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api/customer-attributes/customersattributespost) **BEFORE** creating a customer.
         * attribute_id
         * attribute_value -- This is input as a string, regardless of the [Type](/api-reference/customer-subscribers/v3-customers-api/models/type).
+
+        **Notes**
+        * channel_ids won't be use if the channel listed in origin_channel_id allows global access
```

Comment:
* channel_ids won't be used if the channel listed in origin_channel_id allows global access.

Reply:
@traci707 Do you think we should extend on this note?
Maybe add examples too... so it's clearer how to use it?